lib/src/command/run.dart

```diff
@@ -29,10 +29,15 @@ class RunCommand extends PubCommand {
     argParser.addFlag('enable-asserts', help: 'Enable assert statements.');
     argParser.addFlag('checked', abbr: 'c', hide: true);
     argParser.addOption('mode', help: 'Deprecated option', hide: true);
+    // mode exposed for `dartdev run` to use as subprocess.
```

Comment:
Can we find a more descriptive name? Maybe `--as-subprocess` or something?

Reply:
Or we could say `--dartdev-mode`